app/controllers/katello/api/v2/module_streams_controller.rb

```diff
@@ -3,8 +3,43 @@ class Api::V2::ModuleStreamsController < Api::V2::ApiController
     apipie_concern_subst(:a_resource => N_("a module stream"), :resource => "module_streams")
     include Katello::Concerns::Api::V2::RepositoryContentController
 
+    before_action :check_hosts, :only => :index
+
+    api :GET, "/module_streams", N_("List packages")
+    api :GET, "/content_views/:content_view_id/filters/:filter_id/:resource_id", N_("List :resource_id")
+    api :GET, "/content_view_filters/:content_view_filter_id/:resource_id", N_("List :resource_id")
+    api :GET, "/repositories/:repository_id/:resource_id", N_("List :resource_id")
```

Comment:
I suppose since you are overriding the parent class' index method, you need to redeclare the api docs?

Reply:
I need it to add host_ids